packages/nameguard-js/tsup.config.js

```diff
@@ -8,4 +8,5 @@ export default defineConfig({
   clean: true,
   bundle: true,
   dts: true,
+  external: ["@namehash/nameguard"],
```

Comment:
Interesting. I assumed we wouldn't need to specify anything explicit as `external`? As I understand any dependencies in package.json are automatically excluded? https://tsup.egoist.dev/#excluding-packages